Module not found

Hi Adam,

I am getting this error when i try to just
`import { withData } from 'next-apollo'`

```
./node_modules/next-apollo/dist/link.js
Module not found: Can't resolve 'next-server/dist/lib/utils' in '/Users/***/node_modules/next-apollo/dist'
```

These are my dependencies:
```
"dependencies": {
    "@apollo/react-hooks": "^3.1.1",
    "@apollo/react-ssr": "^3.1.1",
    "@zeit/next-css": "^1.0.1",
    "antd": "^3.9.2",
    "apollo-boost": "^0.4.4",
    "babel-plugin-import": "^1.9.1",
    "graphql": "^14.5.6",
    "next": "latest",
    "next-apollo": "^3.1.4",
    "null-loader": "2.0.0",
    "react": "^16.7.0",
    "react-dom": "^16.7.0"
  }
```

--- a/package.json
+++ b/package.json
@@ -38,7 +38,7 @@
     "@apollo/react-ssr": "^3.0.1",
     "apollo-boost": "^0.4.4",
     "graphql": "^14.3.1",
-    "next": "^9.0.5",
+    "next": "^9.0.6",
     "react-dom": "^16.8.6",
     "react": "^16.8.6"
   },
@@ -57,7 +57,7 @@
     "graphql": "^14.3.1",
     "husky": "0.14.3",
     "lint-staged": "4.3.0",
-    "next": "^9.0.5",
+    "next": "^9.0.6",
     "prettier": "1.15.3",
     "react": "16.8.6",
     "react-dom": "16.8.6"

--- a/src/link.js
+++ b/src/link.js
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types'
 import Link from 'next/link'
 import Router from 'next/router'
 import { getDataFromTree } from '@apollo/react-ssr'
-import { execOnce } from 'next-server/dist/lib/utils'
+import { execOnce } from 'next/dist/next-server/lib/utils'
 import exact from 'prop-types-exact'
 import { format, resolve, parse } from 'url'
 

